ironfish/src/primitives/noteEncrypted.ts

```diff
@@ -15,6 +15,14 @@ export type NoteEncryptedHash = Buffer
 export type SerializedNoteEncryptedHash = Buffer
 export type SerializedNoteEncrypted = Buffer
 
+const ensureBuffer = (value: Buffer | string): Buffer => {
+  if (typeof value === 'string') {
+    return Buffer.from(value, 'hex')
+  } else {
+    return value
+  }
+}
```

Comment:
```suggestion
const ensureBuffer = (value: Buffer | string): Buffer => typeof value === 'string' ? Buffer.from(value, 'hex') : value;
```

More succinct way? totally a preference

Reply:
Thanks. I tend to like more multi-line functions, as I find them much easier to read than one-liners